docs/tutorials/buttons.md

```diff
@@ -0,0 +1,115 @@
+---
+title: Using the Buttons
+subject: Buttons
+---
+
+* Table of Contents
+{:toc}
+
+![EV3 Buttons](/images/lego_ev3_buttons.jpg)
```

Comment:
I don't want to use anything without asking first. As an alternative, @G33kDude, has an svg of the EV3 that he made. We could probably make a similar image that looks even better than the current one.